modules/fts/pages/fts-advanced-default-field.adoc

```diff
@@ -0,0 +1,3 @@
+= Default Field
+The default field for this bucket.
+the default value is `_all`.
```

Comment:
The default field is the field name given to the composite field where the resident content can be searched for by a query that isn't field scoped. Indexed fields will need to have this option checked to support this - "include in _all"